Makefile

```diff
@@ -337,6 +337,12 @@ ifneq ($(filter ppc64%,$(UNAME_M)),)
 	endif
```

Comment:
I'm a bit unsure if this is really used by someone, so I added an additional section specific for ppc64le